keyboards/paladin64/readme.md

```diff
@@ -0,0 +1,19 @@
+# Paladin64 ALPS PCB by /u/iNachie
+
+Firmware for the Paladin64 ALPS64 PCB, with underglow and an optional
+trackpoint module.
+
+Keyboard Maintainer: [/u/iNachie](https://github.com/nachie)  
+Hardware Supported: Paladin64 ALPS PCB by iNachie
+Hardware Availability: [/u/iNachie](https://www.reddit.com/user/inachie/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make paladin64:default
+
+Or to make and flash:
+
+    make obelus:default:dfu
```

Comment:
Would you mind updating this command as well?

Reply:
ah, copy paste ;)